src/components/nav-bar/NavBar.styles.ts

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 
 export const Container = styled.header`
   display: flex;
+  width: 23.4375rem;
```

Comment:
```suggestion
width: 100%;
max-width: 23.4375rem;
```
작은 폰에서 혹시 몰라서 제안해봅니다

Reply:
작은 폰도 font-size 조정이 들어가긴 할 거라서 괜찮을 거 같아요!